test/llvm/with-opaque-ptrs/SKIPIF

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+# Skip versions of LLVM in which we don't have opaque pointers.
+
+import os
+
+print(int(os.getenv('CHPL_LLVM_VERSION')) < 15)
```

Comment:
Ah, I was so confused why it wasn't working -- maybe I should note that in https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/TestSystem.rst.
I will change that and remove the symlinks. As a side note, the reason I ended up using a script skipif is because the simple form doesn't allow inequality conditions, and the list of LLVM versions for which we would like to skip typed pointer tests will grow each new release.

Reply:
Not sure what you meant about the symlinks. The comparison in Python here sounds good. (Python also has a version comparison utility that we use sometimes).

Reply:
I symlinked this SKIPIF into each of the subdirectories to get them all to skip, so if renaming it to `with-opaque-ptrs.skipif` was sufficient to skip the subdirectories I could remove those extra symlinks.

However, I just tried out the renaming and it seems not to skip the subdirectories...

Reply:
It needs to be at the same level as the directory, so you have `with-opaque-ptrs/` next to `with-opaque-ptrs.skipif`. Also if you are checking with `start_test` you need to use `-respect-skipifs` (otherwise the default is to run whatever was on the command line without checking). I don't think even `respect-skipifs` will consider skipifs in parent directories, so when testing the easiest thing to do is probably `start_test test/llvm` and watch what gets run and what gets skipped.

Reply:
Got it, working properly now